Stonks/obj/project.nuget.cache

```diff
@@ -1,14 +1,14 @@
 {
   "version": 2,
-  "dgSpecHash": "y6xN0JaWmSbpOA6qIQ/9eDl1PsgwsRhpzVhjVh8N6PWvEwYQqILLa3HcV0nltSjfq13A+YkAUceXNd7Iup5O8g==",
+  "dgSpecHash": "wVTgpx5YoSflvGvomT5qK5f2cyoKr1mrjI+56NAXS3G1ByHvfNBCJ47LCK1mKZ2kQZg2YHCBoXbHfNT4zcNaEA==",
   "success": true,
-  "projectFilePath": "C:\\Users\\mjaka\\OneDrive\\Documents\\GitHub\\stonks\\Stonks\\Stonks.csproj",
+  "projectFilePath": "C:\\Users\\vartotojas\\Source\\Repos\\stonks\\Stonks\\Stonks.csproj",
   "expectedPackageFiles": [
-    "C:\\Users\\mjaka\\.nuget\\packages\\fontawesome.sharp\\5.14.1\\fontawesome.sharp.5.14.1.nupkg.sha512",
-    "C:\\Users\\mjaka\\.nuget\\packages\\livecharts\\0.9.7\\livecharts.0.9.7.nupkg.sha512",
-    "C:\\Users\\mjaka\\.nuget\\packages\\livecharts.winforms\\0.9.7.1\\livecharts.winforms.0.9.7.1.nupkg.sha512",
-    "C:\\Users\\mjaka\\.nuget\\packages\\livecharts.wpf\\0.9.7\\livecharts.wpf.0.9.7.nupkg.sha512",
-    "C:\\Users\\mjaka\\.nuget\\packages\\modernui\\2.0.0\\modernui.2.0.0.nupkg.sha512"
+    "C:\\Users\\vartotojas\\.nuget\\packages\\fontawesome.sharp\\5.14.1\\fontawesome.sharp.5.14.1.nupkg.sha512",
+    "C:\\Users\\vartotojas\\.nuget\\packages\\livecharts\\0.9.7\\livecharts.0.9.7.nupkg.sha512",
+    "C:\\Users\\vartotojas\\.nuget\\packages\\livecharts.winforms\\0.9.7.1\\livecharts.winforms.0.9.7.1.nupkg.sha512",
+    "C:\\Users\\vartotojas\\.nuget\\packages\\livecharts.wpf\\0.9.7\\livecharts.wpf.0.9.7.nupkg.sha512",
+    "C:\\Users\\vartotojas\\.nuget\\packages\\modernui\\2.0.0\\modernui.2.0.0.nupkg.sha512"
```

Comment:
Gosh, how to avoid these changes?? :D I am confused, I thought .gitignore was the solution.